oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/type/TopNEntity.java

```diff
@@ -38,6 +38,6 @@ public TopNEntity() {
     public TopNEntity(SelectedRecord record) {
         this.name = record.getName();
         this.id = record.getId();
-        this.value = Long.parseLong(record.getValue());
+        this.value = Double.valueOf(record.getValue()).longValue();
```

Comment:
The codes below returns something like "4010.00" even if the value is a `bigint` in MySQL, causing `Long.parseLong` to throw exception

https://github.com/apache/skywalking/blob/0e4e4b888f0f1edac5ab6ae5bef5e860aa14f20b/oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2AggregationQueryDAO.java#L82